astro/src/content/quickstarts/_run-fusionauth-other.mdx

```diff
@@ -0,0 +1,7 @@
+If you plan on running FusionAuth another way, check out our [downloads page](/download). When you start FusionAuth for the first time, go through the First Time Setup wizard - you'll see it at the top of the main Dashboard page. At the end of First Time Setup, you'll find the information you need for the quickstart configuration on the Summary page at the end of the wizard. You'll need to:
+1. Take those values and use them in the `.env` config file
```

Comment:
config is in the knownwords.txt so I am not sure what vale's problem is

Reply:
sometimes i think vale's check must be inverted, it complains about so much :-)